test/ci-integration/cluster/cluster_test.go

```diff
@@ -1,4 +1,4 @@
-// +build e2e
+// +build ci_integration
```

Comment:
Since eventually `clusterctl` will be going away, it might be better to leave this in test/integration, tag this as integration, and retag the clusterctl "integration" test as `clusterctl-integration` instead.

Reply:
That sounds good to me, I think we can even remove the build tag from `clusterctl`?

Reply:
Fixed as we discussed :)